app/assets/src/components/views/SampleUploadFlow/host_organism_message.scss

```diff
@@ -15,12 +17,25 @@
 .messageLine {
   margin: 0 $space-s;
   padding: $space-s;
-
-  &:nth-child(1) {
-    margin-top: $space-s;
-  }
+  font-size: $font-size;
+  margin: 0 44px; // align to right of icon
 
   &:nth-child(odd) {
     background-color: $info-dark;
   }
+
+  &:nth-child(even) {
+    background-color: $info-bg;
+  }
+
+  &:last-child {
+    margin-bottom: $space-m;
+  }
+}
+
+// NOTE: we should really have a Link component
```

Comment:
We do have one... If does not satisfy your needs you should either make it more generic/configurable or add a new one.

Reply:
Which link component are you referring to? `ExternalLink`? 

In any case, the common property that would be valuable is the color, and it's not clear what the generic color should be. There appear to be several shades of blue in use, and there is a global override to force no color on links. 

I'll remove the comment as it may be premature.

Reply:
SGTM, my comment was mostly triggered by the comment.